### ISSUE TITLE
Console apps should be accessible with Talkback

### ISSUE THREAD

**Issue by JanuszChmiel** [jmpessoa/lazandroidmodulewizard#57]
Whe have professional solution, which is capable to create nicely looked applications. But why console applications which uses commands such as writeln or readln can not be made accessible also with Talkback?
So also those apps would be compiled to standard.apk file and could be also executed from a Android launcher such as GUI apps.
I only do not know, if is it technically possible. How much programmers efford would it require to achieve this task.
